spec/controllers/cloud_subnet_controller_spec.rb

```diff
@@ -127,7 +127,7 @@
     before { stub_user(:features => :all) }
 
     it "builds create screen" do
-      post :button, :params => { :pressed => "cloud_subnetnew", :format => :js }
+      post :button, :params => { :pressed => "cloud_subnet_new", :format => :js }
 
```

Comment:
I have doubts about usefullness of this test. The button id was wrong and all was green right? We should probably test something positive, such as partal rendered at least.

Reply:
Yes, the button id was wrong and all was green.